docs/admin/config.rst

```diff
@@ -725,6 +725,63 @@ GET_HELP_URL
 
 URL where support for your Weblate instance can be found.
 
+.. setting:: GITEA_CREDENTIALS
+
+GITEA_CREDENTIALS
+------------------
+
+.. versionadded:: 4.3
+
+List for credentials for Gitea servers.
+
+.. hint::
+
+    Use this in case you want Weblate to interact with more of them, for single
+    Gitea endpoint stick with :setting:`GITEA_USERNAME` and :setting:`GITEA_TOKEN`.
+
+.. code-block:: python
+
+    GITHUB_CREDENTIALS = {
+        "api.github.com": {
+            "username": "weblate",
+            "token": "your-api-token",
+        },
+        "github.example.com": {
+            "username": "weblate",
+            "token": "another-api-token",
+        },
+    }
+
+.. setting:: GITEA_USERNAME
+
+GITEA_USERNAME
+---------------
+
+Gitea username used to send pull requests for translation updates.
+
+.. seealso::
+
+   :setting:`GITEA_CREDENTIALS`,
+   :ref:`vcs-gitea`
+
+.. setting:: GITEA_TOKEN
+
+GITEA_TOKEN
+------------
+
+.. versionadded:: 4.3
+
+Gitea personal access token used to make API calls to send pull requests for
+translation updates.
+
+.. seealso::
+
+   :setting:`GITEA_CREDENTIALS`,
+   :ref:`vcs-gitea`,
```

Comment:
The target for this is missing, the documentation for Gitea should be in `docs/vcs.rst`.